starkinfra/individualidentity/__individualidentity.py

```diff
@@ -0,0 +1,150 @@
+from ..utils import rest
+from starkcore.utils.resource import Resource
+from starkcore.utils.checks import check_datetime, check_date
+
+
+class IndividualIdentity(Resource):
+    """# IndividualIdentity object
+    IndividualIdentities are used to aggregate an object list of IndividualDocuments 
+    for natural person documents check.
+    When you initialize a IndividualIdentity, the entity will not be automatically
```

Comment:
An IndividualDocument represents an individual to be validated. It can have several individual documents attached to it, which are used to validade the identity of the individual. Once an individual identity is created, individual documents must be attached to it using the created method of the individual document resource. When all the required individual documents are attached to an individual identity it can be sent to validation by patching its status to processing.